lib/miq_automation_engine/engine/miq_ae_service_model_base.rb

```diff
@@ -109,12 +109,12 @@ def self.wrap_results(results)
         if results.nil?
           ret = nil
         elsif results.kind_of?(Array)
-          ret = drb_return(results.collect { |r| wrap_results(r) })
+          ret = results.collect { |r| wrap_results(r) }
```

Comment:
I think wrap_results can just go away, because its only purpose (I think) was to facilitate the drb_return stuff.